File: hamersapp/src/main/java/nl/ecci/hamers/beers/Review.java
package nl.ecci.hamers.beers;

import com.google.gson.annotations.SerializedName;

import java.util.Date;

class Review {

    public static final String REVIEW = "REVIEW";

    private final int id;
    @SerializedName("beer_id")
    private final int beerID;
    @SerializedName("user_id")
    private final int userID;
    private final String description;
    private final int rating;
    private final Date proefdatum;
    @SerializedName("created_at")
    private final Date createdAt;

    public Review(int id, int beerID, int userID, String description, int rating, Date createdAt, Date proefdatum) {
        super();
        this.id = id;
        this.beerID = beerID;
        this.userID = userID;
        this.description = description;
        this.rating = rating;
        this.createdAt = createdAt;
        this.proefdatum = proefdatum;
    }

    public int getID() {
        return id;
    }

    public int getBeerID() {
        return beerID;
    }

    public int getUserID() {
        return userID;
    }

    public String getDescription() {
        return description;
    }

    public int getRating() {
        return rating;
    }

    public Date getCreatedAt() {
        return createdAt;
    }

    public Date getProefdatum() {
        return proefdatum;
    }
}

File: hamersapp/src/main/java/nl/ecci/hamers/beers/Review.kt
package nl.ecci.hamers.beers

import com.google.gson.annotations.SerializedName
import nl.ecci.hamers.helpers.Utils
import java.util.*

class Review(val id: Int = Utils.notFound,
             @SerializedName("beer_id")
             val beerID: Int = Utils.notFound,
             @SerializedName("user_id")
             val userID: Int = Utils.notFound,
             val description: String = Utils.unknown,
             val rating: Int = Utils.notFound,
             @SerializedName("created_at")
             val createdAt: Date = Date(),
             val proefdatum: Date = Date()) {

    companion object {
        val REVIEW = "REVIEW"
    }
}